content/docs/release-notes/archive/2107-1-release-notes.md

```diff
@@ -1,14 +1,12 @@
 ---
 linktitle: 2107.1 Release Notes
 title: Release Notes for Open 3D Engine 2107.1 (Developer Preview)
-description: Find out what's was delivered in O3DE 2107.1 (Developer Preview).
-weight: 101
+description: Find out what was delivered in O3DE 2107.1 (Developer Preview).
+weight: 102
```

Comment:
Assuming release notes will be listed in reverse chronological order of release, this first release should be weighted such that its weight doesn't need to be edited every release.
```suggestion
weight: 200
```